test/functional/testlib.bash

```diff
@@ -543,6 +543,79 @@ remove_from_manifest() {
 
 }
 
+# Updates fields in an existing manifest
+# Parameters:
+# - -p: if the p (partial) flag is set the function skips updating the hashes
+#       in the MoM, this is useful if more changes are to be done in order to
+#       reduce time
+# - MANIFEST: the relative path to the manifest file
+# - KEY: the thing to be updated
+# - HASH/NAME: the file name or hash of the record to be updated (if applicable)
+# - VALUE: the value to be used for updating the record
+update_manifest() {
```

Comment:
Awesome addition! update readme?

Reply:
This one is a good candidate for adding to the readme, I'll update it.